src/atom-environment.js

```diff
@@ -1306,12 +1300,8 @@ class AtomEnvironment {
   }
 
   setRepresentedFilename(filename) {
-    if (
-      typeof this.applicationDelegate.setWindowRepresentedFilename ===
-      'function'
-    ) {
-      this.applicationDelegate.setWindowRepresentedFilename(filename);
-    }
+    // eslint-disable-next-line no-unused-expressions
+    this.applicationDelegate.setWindowRepresentedFilename?.(filename);
```

Comment:
this is not as safe as before, if `setWindowRepresentedFilename` isn't a function this will rise an error.

Reply:
I don't think that is accurate. `?.` can also be used for functions.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls

Reply:
Yes but if is existing and not a function will throw an error.
![image](https://user-images.githubusercontent.com/1194048/113413518-2c84cc00-93bb-11eb-9a2a-005e347dac83.png)
I was just pointing out that this is not a syntax only change but a behavioural change.